src/Sentry.AspNetCore/Extensions/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Text;
+
+namespace Sentry.AspNetCore.Extensions
+{
+    internal static class StringBuilderExtensions
+    {
+        internal static StringBuilder AppendIf(this StringBuilder builder, bool condition, string? value)
```

Comment:
You want to append null? If you have a condition already here doesn't it make sense to have `string value`?